lib/resolvers.js

```diff
@@ -38,6 +70,56 @@ const Resolvers = {
     }
   },
   Mutation: {
+    createAccount: async (root, { id, isAdmin }, request) => {
+      await validateAdminAccess(request);
+      const sql = 'CALL create_account(?, ?)';
+      const pool = request.server.app.mysql;
+      const results = await Common.doQuery(sql, [id, isAdmin ? 1 : 0], pool);
+
+      // Sanity check. This should always be one.
+      if (results.affectedRows !== 1) {
```

Comment:
At some point we should replace all of these sanity check `if` statements with assertions.